zabbix-java-gateway/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+logfile=/var/log/zabbix/zabbix_java_gateway.log
+logdir=$(dirname $logfile)
+pidfile=/var/run/zabbix/zabbix_java_gateway.pid
+piddir=$(dirname $pidfile)
+mkdir -p $piddir $logdir
+touch $logfile
+chown -R zabbix:zabbix $piddir $logfile
+/etc/init.d/zabbix-java-gateway start
+sleep 1
+tail -f $logfile --pid=$(cat $pidfile)
```

Comment:
I think we can relax it to 3 seconds. Or better, use a while loop until the
pid file exists.

On Mon, Mar 16, 2015 at 10:04 PM, Viktor Shlapakov <notifications@github.com
> wrote:

> In zabbix-java-gateway/run.sh
> <https://github.com/scrapinghub/docker-images/pull/5#discussion_r26486056>
> :
>
> > @@ -0,0 +1,11 @@
> > +#!/bin/bash
> > +logfile=/var/log/zabbix/zabbix_java_gateway.log
> > +logdir=$(dirname $logfile)
> > +pidfile=/var/run/zabbix/zabbix_java_gateway.pid
> > +piddir=$(dirname $pidfile)
> > +mkdir -p $piddir $logdir
> > +touch $logfile
> > +chown -R zabbix:zabbix $piddir $logfile
> > +/etc/init.d/zabbix-java-gateway start
> > +sleep 1
> > +tail -f $logfile --pid=$(cat $pidfile)
>
> @lins05 <https://github.com/lins05> it worked for me in most cases, but
> your question is reasonable, what is a better value from your point of
> view? 3s/5s?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scrapinghub/docker-images/pull/5/files#r26486056>.
>